### PR TITLE
New Post: Model Cards 

### PR DESCRIPTION
**Pull request by EziOzoani**
there are some additions to this blog post still outstanding:
- linking to mentioned sections in the guidebook( based on this [PR](https://github.com/huggingface/hub-docs/pull/545))
- one final image 

Inputs welcomed 🤗

### CODE REVIEW

File: model-cards.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Model Cards" 
+thumbnail: /blog/assets/121_model-cards/thumbnail.png
```
Comment (julien-c) [inline model-cards.md:3]: you should merge the two subfolders of assets that are in this PR

File: model-cards.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
```
Comment (julien-c) [inline model-cards.md:1]: the filename for this file will end up being the url of the post under hf.co/blog, should we pick something more specific?

File: model-cards.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+Let's look ahead
+<p align="center">
+  <img src="assets/121_model-cards/looking_ahead.png" width="250"/>
+</p>
+
+
+As model cards continue to grow and evolve within the ML, ethics, policy and further spaces, so will this tool.
+
+* The Hugging Face ecosystem will continue to advance methods that streamline Model Card creation [through code](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool) and user interfaces.
```
Comment (julien-c) [inline model-cards.md:102]:
```suggestion
* The Hugging Face ecosystem will continue to advance methods that streamline Model Card creation [through code](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool) and user interfaces, including building more features directly into the repos and product. 
```

maybe hint at something like this? (not sure)

File: model-cards.md (hunk 1 above)
Comment (EziOzoani) [reply model-cards.md:3]: done

File: model-cards.md (hunk 2 above)
Comment (EziOzoani) [reply model-cards.md:1]: @meg-huggingface thoughts on:

- model-cards-launch
- v1-model-cards
or any other suggestions

File: model-cards-launch.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+### Standardising Model Card Structure
+Through our background research and user studies, we aimed to establish a standard of "model cards" as understood by the general public. From this research, we found that there is a variance in the content of a model card and the layout of a model card - the presence or absence of some headings and sub-headings - but there was consistency in the different types of headings.
+
+Through our user studies and as  illustrated in the model book **[link to section in guide book]**, we noted that  while the Bias, Risks and Limitations section ranks second for both model card writers and model card readers for In what order do you write the model card and What section do you look at first, respectively, it is also noted as the most challenging/longest section to write.
```
Comment (osanseviero) [inline model-cards-launch.md:66]:
```suggestion
Through our user studies and as illustrated in the model book **[link to section in guide book]**, we noted that while the **Bias, Risks, and Limitations** section ranks second for both model card writers and model card readers for In what order do you write the model card and What section do you look at first, respectively, it is also noted as the most challenging/longest section to write. 
```

Comment (osanseviero) [inline model-cards-launch.md:66]: I found this sentence a bit challenging to read. Not sure how yet, but maybe we could reword it a bit :thinking:

File: model-cards-launch.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+### Standardising Model Card Structure
+Through our background research and user studies, we aimed to establish a standard of "model cards" as understood by the general public. From this research, we found that there is a variance in the content of a model card and the layout of a model card - the presence or absence of some headings and sub-headings - but there was consistency in the different types of headings.
+
+Through our user studies and as  illustrated in the model book **[link to section in guide book]**, we noted that  while the Bias, Risks and Limitations section ranks second for both model card writers and model card readers for In what order do you write the model card and What section do you look at first, respectively, it is also noted as the most challenging/longest section to write. 
+
+Directed by these findings we created a  research informed  model card template, that not only standardised the structure and content of HF model cards but also provided default prompted text, with the aim  to aid with writing model card sections, with a particular focus on the Bias, Risks and Limitations section.
```
Comment (osanseviero) [inline model-cards-launch.md:68]:
```suggestion
Directed by these findings, we created a research-informed model card template that not only standardized the structure and content of HF model cards but also provided default prompted text, with the aim to aid with writing model card sections, with a particular focus on the Bias, Risks and Limitations section.
```

Comment (osanseviero) [inline model-cards-launch.md:68]: Should we link to the template here?

File: model-cards-launch.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+
+### Accessibility and Inclusion
+
+In order to lower barriers to entry for creating model cards, we designed [the model card writing tool](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool), a tool with a graphical user interface (GUI) to enable people and teams with different skill sets and roles to easily collaborate and create model cards, without needing to code or use markdown.
```
Comment (osanseviero) [inline model-cards-launch.md:73]: Should we also mention https://github.com/huggingface/moon-landing/pull/4902 as that will be built into the product?

File: model-cards-launch.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+### Accessibility and Inclusion
+
+In order to lower barriers to entry for creating model cards, we designed [the model card writing tool](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool), a tool with a graphical user interface (GUI) to enable people and teams with different skill sets and roles to easily collaborate and create model cards, without needing to code or use markdown. 
+
+<p align="center">
+  <img src="assets/121_model-cards/upload_a_mc.gif" width="800"/>
+</p>
+
+
+The writing tool encourages those who have yet to write model cards to create them more easily and those who have previously written them, this approach invites them to add to the prompted information. All while centring the ethical components of model documentation.
```
Comment (osanseviero) [inline model-cards-launch.md:80]:
```suggestion
The writing tool encourages those who have yet to write model cards to create them more easily and those who have previously written them. This approach invites them to add to the prompted information. All while centering the ethical components of model documentation.
```

File: model-cards-launch.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+Let's look ahead
+<p align="center">
+  <img src="assets/121_model-cards/looking_ahead.png" width="250"/>
+</p>
+
+
+As model cards continue to grow and evolve within the ML, ethics, policy and further spaces, so will this tool.
+
+* The Hugging Face ecosystem will continue to advance methods that streamline Model Card creation [through code](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool) and user interfaces, including building more features directly into the repos and product.
```
Comment (osanseviero) [inline model-cards-launch.md:102]: The first link shows how to do it with user interfaces. For the "through code" maybe we could share https://huggingface.co/docs/huggingface_hub/how-to-model-cards ?

File: model-cards-launch.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+Let's look ahead
+<p align="center">
+  <img src="assets/121_model-cards/looking_ahead.png" width="250"/>
+</p>
+
+
+As model cards continue to grow and evolve within the ML, ethics, policy and further spaces, so will this tool.
+
+* The Hugging Face ecosystem will continue to advance methods that streamline Model Card creation [through code](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool) and user interfaces, including building more features directly into the repos and product. 
+* As we further develop model tools such as [Evaluate on the Hub](https://huggingface.co/blog/eval-on-the-hub), we will integrate their usage within the model card development workflow. For example, as automatically evaluating model performance across disaggregated factors becomes easier, these results will be possible to import into the model card.
```
Comment (osanseviero) [inline model-cards-launch.md:103]:
```suggestion
* As we further develop model tools such as [Evaluation on the Hub](https://huggingface.co/blog/eval-on-the-hub), we will integrate their usage within the model card development workflow. For example, as automatically evaluating model performance across disaggregated factors becomes easier, these results will be possible to import into the model card.
```

**Review by osanseviero — COMMENTED** [review]
Very cool! I did a first pass through it

File: model-cards-launch.md (hunk 6 above)
Comment (EziOzoani) [reply model-cards-launch.md:74]: using this direct link or is there a different one that you would prefer?

File: model-cards-launch.md (hunk 5 above)
Comment (EziOzoani) [reply model-cards-launch.md]: that's the aim but see updates in [this thread ](https://huggingface.slack.com/archives/C02GLJ5S0E9/p1671468888434239?thread_ts=1671055278.964029&cid=C02GLJ5S0E9)

File: model-cards-launch.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
+---
```
[the remark this answers is not in the document]
Comment (julien-c) [reply model-cards-launch.md:1]: see https://github.com/huggingface/blog/pull/719#discussion_r1047804416 :)

File: _blog.yml [hunk 11]
```diff
@@ -1570,3 +1571,15 @@
   date: Dec 15, 2022
   tags:
     - ethics
+
+
+- local: model-cards
+  title: "Model Cards: Introducing HF Model docuemtntion tools"
```
Comment (meg-huggingface) [inline _blog.yml:1577]:
```suggestion
  title: "Model Cards: Introducing HF Model documentation tools"
```

File: model-cards-launch.md [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+            <code>Marissa Gerchick</code>
+            <span class="fullname">Marissa Gerchick</span>
+        </div>
+    </a>
+</div>
+
+
+<div class="author-card">
+    <a href="/Meg"> 
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1626214544196-60c757ea5f9a76ab3f844f12.png?w=200&h=200&f=face" width=100 title="Gravatar">
```
Comment (meg-huggingface) [inline model-cards-launch.md:38]:
```suggestion
        <img class="avatar avatar-user" src="https://avatars.githubusercontent.com/u/90473723" width=100 title="Gravatar">
```

File: model-cards-launch.md [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+            <span class="fullname">Margaret Mitchell</span>
+        </div>
+    </a>
+</div>
+
+## Introduction
+
+Model cards are an important documentation framework for understanding, sharing, and improving machine learning models. When done well, a model card can serve as a _boundary object_, a single artifact accessible to people with different backgrounds and goals in understanding models - including developers, students, policymakers, ethicists, and those impacted by machine learning models.
+
+Today, we launch a [model card creation tool](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool) and a model card Guide Book, which details how to fill out model cards, user studies, and state of the art in ML documentation. This work, building from many other people and organizations, focuses on the _inclusion_ of people with different backgrounds and roles. We hope it serves as a stepping stone in the path toward improved ML documentation.
```
Comment (meg-huggingface) [inline model-cards-launch.md:50]:
```suggestion
Today, we launch a [model card creation tool](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool) and [a model card Guide Book](https://huggingface.co/docs/hub/model-card-guidebook.md), which details how to fill out model cards, user studies, and state of the art in ML documentation. This work, building from many other people and organizations, focuses on the _inclusion_ of people with different backgrounds and roles. We hope it serves as a stepping stone in the path toward improved ML documentation.
```

**Pull request merged**